<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Mocha Tests" time="279.68" tests="1" failures="2">
  <testsuite name="Root Suite" timestamp="2019-09-22T11:13:23" tests="0" failures="0" time="0">
  </testsuite>
  <testsuite name="Credit Card 3-D Secure test" timestamp="2019-09-22T11:13:23" tests="1" file="/home/travis/shopware/custom/plugins/WirecardElasticEngine/Tests/Selenium/Payments/CreditCardThreeDTest.js" failures="2" time="279.68">
    <testcase name="Credit Card 3-D Secure test should check the credit card 3ds payment process" time="152.451" classname="should check the credit card 3ds payment process">
      <failure message="ETIMEDOUT connect ETIMEDOUT 207.254.8.4:80" type="Error"><![CDATA[Error: ETIMEDOUT connect ETIMEDOUT 207.254.8.4:80
    at ClientRequest.<anonymous> (node_modules/selenium-webdriver/http/index.js:258:15)
    at Socket.socketErrorListener (_http_client.js:387:9)
    at emitErrorNT (internal/streams/destroy.js:64:8)
    at _combinedTickCallback (internal/process/next_tick.js:138:11)
    at process._tickCallback (internal/process/next_tick.js:180:9)]]></failure>
    </testcase>
    <testcase name="Credit Card 3-D Secure test &quot;after all&quot; hook" time="127.229" classname="&quot;after all&quot; hook">
      <failure message="ETIMEDOUT connect ETIMEDOUT 207.254.8.4:80" type="Error"><![CDATA[Error: ETIMEDOUT connect ETIMEDOUT 207.254.8.4:80
    at ClientRequest.<anonymous> (node_modules/selenium-webdriver/http/index.js:258:15)
    at Socket.socketErrorListener (_http_client.js:387:9)
    at emitErrorNT (internal/streams/destroy.js:64:8)
    at _combinedTickCallback (internal/process/next_tick.js:138:11)
    at process._tickCallback (internal/process/next_tick.js:180:9)]]></failure>
    </testcase>
  </testsuite>
</testsuites>